### ISSUE TITLE
use pagination to speed up

### ISSUE THREAD

**Issue by gouchaoer** [erikdubbelboer/phpRedisAdmin#92]
if there is thousands of keys in redis, web frontend will load all data from backend, which is slow and stupid.

1,use pagination on keys which have the same prefix
2,actually user do not need to load all keys  on the left side, so only load keys that user choose. 

phpredisadmin is difficult to use in develop environment because it load all keys!

**Comment by erikdubbelboer** [comment]
Before when we used the `KEYS` command pagination was never really possible. Now with the `SCAN` command it is. But a lot needs to be modified for this to work and at the moment I don't have time for this I'm afraid.

**Comment by gouchaoer** [comment]
I suggest that anyone who uses phpredisadmin to visualize redis in development enviroment, do not set too much keys in global set. You can always use a hash table to do it except that you want to set an expire time.

**Comment by GregOriol** [comment]
phpredisadmin is completely broken for me on a setup where there are >100'000 keys :-(

**Comment by sombahadurlimbu** [comment]
Without pagination, its difficult to work on large set of keys. Any news when this will be included? Also in case of clusters, rather than searching keys or keys pattern to individual redis servers, a common interface to  search keys for the cluster nodes will be great.

**Comment by gouchaoer** [comment]
@sombahadurlimbu if yo find that you need to find keys in redis, you probably need rational database such as mysql(for large scale maybe tidb). redis is a cache and we should not query or iterate things in redis.

**Comment by erikdubbelboer** [comment]
Yes pagination would definitely be a good addition for big data sets.

In theory the javascript code should do ajax requests which result in SCAN operations in redis to fetch all the keys. This way PHP won't run out of memory or time.

But this is quite a big change and I just don't have the time to write this at the moment I'm afraid 😞 

Pull requests are always welcome!
